app/(routes)/pastEvents/[eventId]/photoAlbum/page.tsx

```diff
@@ -1,9 +1,30 @@
-import { NextPageContext } from 'next';
-
 import Wrapper from '@/components/layout/Wrapper';
 import { getEventPhotoAlbum } from '@/app/service/meetup/MeetupService';
+import { Inter } from 'next/font/google'
+import Gallery from '@/components/Gallery/Gallery';
+
+const inter = Inter({ 
+  subsets: ['latin'],
+  weight: ['700'],
+})
```

Comment:
I haven't used Inter before but I'd think you should be able to use Tailwind for all the styling